src/handlers/mentions.rs

```diff
@@ -61,23 +63,42 @@ pub(super) async fn parse_input(
     {
         let file_paths: Vec<_> = files.iter().map(|fd| Path::new(&fd.filename)).collect();
         let to_mention: Vec<_> = config
-            .paths
+            .entries
             .iter()
-            .filter(|(path, MentionsPathConfig { cc, .. })| {
-                let path = Path::new(path);
-                // Only mention matching paths.
-                let touches_relevant_files = file_paths.iter().any(|p| p.starts_with(path));
+            .filter_map(|(entry, MentionsEntryConfig { cc, type_, .. })| {
+                let relevant_file_paths: Vec<PathBuf> = match type_ {
+                    MentionsEntryType::Filename => {
+                        let path = Path::new(entry);
+                        // Only mention matching paths.
+                        file_paths
+                            .iter()
+                            .filter(|p| p.starts_with(path))
+                            .map(|p| PathBuf::from(p))
+                            .collect()
+                    }
+                    MentionsEntryType::Content => {
+                        // Only mentions byte-for-byte matching content inside the patch.
+                        files
+                            .iter()
+                            .filter(|f| f.patch.contains(&**entry))
```

Comment:
Didn't Ralf wish for only checking additions in #1851 (i.e., excluding deletions and context lines)?

Reply:
> Didn't Ralf wish for checking _additions_ only in #1851 (i.e., exclude deletions and context lines)? Tho I guess also flagging _deletions_ might be quite important in some cases.

I missed that, but as you said flagging _deletions_ might also be important. If someone really want that we can IMO see later.

> That leaves us with "context lines"? Does `f.patch` contain these or are we good?

They do contain "context lines". I guess we'll want to restrict to only additions and removal lines.

Reply:
That's done.